app/controllers/books_controller.rb

```diff
@@ -4,11 +4,20 @@ class BooksController < ApiController
   def show
     render json: Book.find(params[:id]), serializer: BookSerializer
   rescue ActiveRecord::RecordNotFound
-    render json: { error: "El libro con el id #{params[:id]} no existe" }, status: :not_found
+    render json: { error: t('book_error', code: params[:id]) }, status: :not_found
   end
 
   def index
-    @books = Book.all
-    render_paginated @books
+    render_paginated Book.all
+  end
+
+  def info_book
+    params.require(:isbn)
```

Comment:
You can create a private method for this param, such as `info_book_params` and then in the OpenLibraryService call, do `OpenLibraryService.new(info_book_params).book_info`